pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -271,7 +307,11 @@ func assignFlavors(log logr.Logger, requests []workload.PodSetResources, podSets
 				}
 				break
 			}
-			flavors, status := assignment.findFlavorForResourceGroup(log, rg, podSet.Requests, resourceFlavors, cq, &podSets[i].Template.Spec)
+			var lastScheduleFlavorInfo map[corev1.ResourceName]int
+			if lastSchedule != nil {
+				lastScheduleFlavorInfo = lastSchedule.LastScheduledFlavorIdx[podSet.Name]
```

Comment:
```suggestion
				lastScheduleFlavorInfo = lastSchedule.LastScheduledFlavorIdx[i]
```

Better to use slices, as they are faster to allocate and query.

Reply:
done, thanks